README.md

```diff
@@ -90,457 +90,467 @@ type: tutorial
 
-![ocp-routes](doc/source/images/ocproutes.png)
+    ![ocp-routes](images/ocproutes.png)
 
-- You can now visit the URL to access the E.D.D.I Dashboard.
+1. You can now visit the URL to access the E.D.D.I Dashboard.
 
-![ocp-eddidashboard](doc/source/images/eddidashboard.png)
+    ![ocp-eddidashboard](images/eddidashboard.png)
 
 ### Step 4: Create a chatbot in E.D.D.I
 
-- Now, let's create a chatbot in E.D.D.I from scratch.
+Now, let's create a chatbot in E.D.D.I from scratch.
+
+To create a chatbot, you will complete these steps: <!--EM: I feel like, at the very least, we need to link people to where they can learn more about these concepts (regular dictionary, behavior rules, output sets). Is this common vocab for developing chatbots? Why would readers need to develop these things? This feels so advanced and I'm worried about just dropping our readers into this without more explanatory text -->
 
-- The steps involved in creating a chatbot are as follows:
-    - Create a `Regular Dictionary`
-    - Create `Behavior Rules`
-    - Create `Output sets`
-    - `Packaging` it
-    - Create a `bot`
+1. Create a regular dictionary
+1. Create behavior rules
+1. Create output sets
+1. Package your dictionary, rules, and sets
+1. Create a chatbot
 
-- In order to build a Chatbot with E.D.D.I, you will have to create a few configuration files and POST them to the corresponding REST APIs. You can use tools like [Postman](https://www.postman.com/downloads/) to make the API calls.
+In order to build a Chatbot with E.D.D.I, you will have to create a few configuration files and POST them to the corresponding REST APIs. You can use tools like [Postman](https://www.postman.com/downloads/) to make the API calls.
 
-#### Step 4.1: Create Regular Dictionary
+#### Step 4.1: Create a regular dictionary
 
-- We create regular dictionaries in order to store custom words and phrases.
+*Regular dictionaries* are used store custom words and phrases. <!--EM: We should clarify this. Is that in machine learning world? AI world?->
 
-- Make a `POST` to `<eddi-url>/regulardictionarystore/regulardictionaries` with a JSON in the body as follows.
+- Make a `POST` to `<eddi-url>/regulardictionarystore/regulardictionaries` with a JSON file in the body as follows. <--EM: Is this what is described below? And "in the body" of what?->
 
->NOTE: You will have the `<eddi-url>` generated in [step 1](#step-1-install-the-eddi-operator-from-red-hat-marketplace-on-openshift-cluster).
+>Note: You will use the `<eddi-url>` generated in [step 1](#-step-1-configure-a-red-hat-openshift-cluster-with-red-hat-marketplace).
 
-- In Postman, enter the `<eddi-url>` followed by `/regulardictionarystore/regulardictionaries`, make sure you select the following parameters:
+In Postman, enter the `<eddi-url>` followed by `/regulardictionarystore/regulardictionaries`. Make sure you select the following parameters: <!--EM: What is Postman? How do they get it? Should that in the prereq?-->
```

Comment:
**What is Postman? How do they get it? Should that in the prereq?**

Postman is a collaboration platform for API development. Its an Open Source tool. Yes, this is a prereq for this tutorial.